src/day8/re_frame/localstorage.cljs

```diff
@@ -0,0 +1,16 @@
+(ns day8.re-frame.trace.localstorage
+  (:require [goog.storage.Storage :as Storage]
+            [goog.storage.mechanism.HTML5LocalStorage :as html5localstore]))
+
+(defn- storage []
+  (let [mech (goog.storage.mechanism.HTML5LocalStorage.)
+        store (goog.storage.Storage. mech)]
+    store))
+
+(defn get! [key]
+  (let [store (storage)]
+    (cljs.reader/read-string (.get store key))))
+
+(defn set! [key val]
+  (let [store (storage)]
+    (.set store key val)))
```

Comment:
You should do the conversion here, rather than getting the client code to do it. Writing EDN on CLJS is a little bit of a tricky story, but I think `pr-str` is probably your friend here.